docs/blog/posts/2024/warp-2-0.md

```diff
@@ -0,0 +1,109 @@
+---
+title: '@warp-ds v2 - DBA Changes Are Here!'
+date: 2024-07-02
+---
+
+DBA beta changes are now available on NPM and EIK.
+
+---
+
+### How to Update Your Apps for DBA
+
+#### Packages to Install from NPM
+
+| Package Name       | Version        |
+| ------------------ | -------------- |
+| `@warp-ds/css`     | `2.0.0-next.1` |
+| `@warp-ds/drive`   | `2.0.0`        |
+| `@warp-ds/react`   | `2.0.0-next.0` |
+| `@warp-ds/vue`     | `2.0.0-next.0` |
```

Comment:
I'm assuming that @warp-ds/vue will get `3.0.0-next.1` and not `2.0.0-next.0`, once we include the breaking changes and make a new release to next? :)  The current version is at `2.0.1-next.4`

Reply:
You're right. Fixed